Source/Core/LMCHFSSResponseFileHandler.cc

```diff
@@ -132,12 +132,15 @@ namespace locust
     fComplexFFT.ApplyWindowFunction(fTFNBins, fTFComplex);
     fComplexFFT.GenerateFIR(fTFNBins,fTFComplex,fFIRComplex);
     fResolution=fComplexFFT.GetTimeResolution();
-
-    for (int i = 0; i < fFIRNBins; ++i)
+    FILE * fFIRout = fopen("output/FIR.txt", "w");
+    fprintf(fFIRout,"#FIR for cavity simulation (index,coefficient)\n");
```

Comment:
Can this comment line be made more general?  I think the file `FIR.txt`  will be written for any TF.  Also, I think it is helpful to output this file routinely.

Reply:
Ok, I adjusted that comment line to remove the explicit cavity reference